f8a_report/report_helper.py

```diff
@@ -838,18 +839,18 @@ def get_report(self, start_date, end_date, frequency='daily', retrain=False):
             worker_result = {}
             if not result_interim:
                 logger.error('No v1 Stack Analyses found in last 1 day.')
-                return worker_result, ingestion_results
+                return worker_result, ingestion_results, missing_latest_node
 
             for worker in worker_list:
                 if worker == 'stack_aggregator_v2':
                     worker_result[worker] = self.create_venus_report(result_interim[worker])
                 # can add results for more workers later
 
-            return worker_result, ingestion_results
+            return worker_result, ingestion_results, missing_latest_node
         else:
```

Comment:
we dont need `else`.

Reply:
This was an existing line in the code. It is used to log a proper message in case of no SA request were done, which I think is should be there. If you still want me to remove else block, I will do it.

Reply:
why do you think we dont need else here?

Reply:
@jparsai Else block is to be there. I'm asking `else` statement is not required. 
See an Example below:
```
def return_even(number):
  if number % 2 == 0:
     return "even"
  return "odd"
```
cc : @yzainee